doc/server/vertx.md

```diff
@@ -181,16 +180,19 @@ object Short extends zio.App {
 
   val attach = VertxZioServerInterpreter().route(responseEndpoint.zServerLogic { key => UIO.succeed(key) })
 
-  override def run(args: List[String]): URIO[ZEnv, ExitCode] =
-    ZManaged.make(ZIO.effect {
-      val vertx = Vertx.vertx()
-      val server = vertx.createHttpServer()
-      val router = Router.router(vertx)
-      attach(router)
-      server.requestHandler(router).listen(8080)
-    } >>= (_.asRIO))({ server =>
-      ZIO.effect(server.close()).flatMap(_.asRIO).orDie
-    }).useForever.as(ExitCode.success).orDie
+  override def run =
+    ZManaged
+      .acquireReleaseWith(ZIO.attempt {
+        val vertx = Vertx.vertx()
+        val server = vertx.createHttpServer()
+        val router = Router.router(vertx)
+        attach(router)
+        server.requestHandler(router).listen(8080)
+      } flatMap (_.asRIO))({ server =>
+        ZIO.attempt(server.close()).flatMap(_.asRIO).orDie
+      })
+      .useForever
+      .exitCode
```

Comment:
The `.exitCode` is no longer required. 👍

Reply:
Removed.